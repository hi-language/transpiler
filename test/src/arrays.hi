items: [10, "hello", !0]
_(items[0])
_(items[1])

items[0] = 20
_(items[0])
