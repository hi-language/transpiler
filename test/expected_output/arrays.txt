10
hello
20
